core/include/detray/propagator/base_stepper.hpp

```diff
@@ -35,6 +35,22 @@ struct void_inspector {
                                                  const char * /*ignored*/) {}
 };
 
+struct config {
```

Comment:
I am not sure if it is right to have rk-specific config in `base_stepper`.

Reply:
No, indeed, I don't like it either, but it is the only way I saw to have a config that does not depend on the stepper type, so that we integrate in traccc code long before we know the type of stepper we are using. If you don't like that, I can change it back to a nested config, but then we will be stuck with:
```
propagator_t::config{}
```
instead of
```
propagation::config{}
```

Reply:
I think it is OK for now